packages/contracts/utils/environment.ts

```diff
@@ -0,0 +1,58 @@
+import {HARDHAT_NETWORK_NAME} from 'hardhat/plugins';
+import {Network} from 'hardhat/types';
+
+/**
+ * Provides default values for environment variables if running against a hardhat node
+ * else requires that the environment variables are set
+ * @param network the hardhat network object
+ * @param name of the environment variable
+ * @param defaultValue the fallback value to be used if not set and in development mode
+ * @returns the value of the environment variable, or a fallback if possible
+ */
+export function env(
+  network: Network,
+  name: string,
+  defaultValue: string
+): string {
+  const value = process.env[name];
+  const isLocal =
+    network.name === HARDHAT_NETWORK_NAME || network.name === 'localhost';
+  if (!isLocal && !value) {
+    throw new Error(`Missing env var: ${name}`);
+  }
+  return process.env[name] || defaultValue;
+}
+
+/// Specific environment variables with hardcoded defaults
+
+export const daoDomainEnv = (network: Network): string =>
+  env(network, `${network.name.toUpperCase()}_DAO_ENS_DOMAIN`, 'dao.eth');
+
+export const pluginDomainEnv = (network: Network): string =>
+  env(network, `${network.name.toUpperCase()}_PLUGIN_ENS_DOMAIN`, 'plugin.eth');
+
+export const managementDaoSubdomainEnv = (network: Network): string =>
+  env(network, 'MANAGEMENT_DAO_SUBDOMAIN', 'management');
+
+export const managementDaoMultisigApproversEnv = (network: Network): string =>
+  env(
+    network,
+    'MANAGEMENT_DAO_MULTISIG_APPROVERS',
+    // random address
+    '0x6B2b5d4F0a40134189330e2d46a9CDD01C01AECD'
```

Comment:
What are the implications of using a random address here?

Instead of a random address, we could think about using the known hardhat default address (derived from hardhat's default mnemonic `'test test test test test test test test test test test junk'`. 
I think it is `0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266`.

This could result in tests passing (that mostly use signers[0]), although this could be equally undesirable.

Reply:
We already use the private key for the HH address you listed as default. So possibly we can use the next derived address from that same mnemonic.

Reply:
Yes, sounds good. Maybe for the default, we can have two or three approvers (and a 1/2 or 2/3 threshold) - all from the default mnemonic.

I am wondering how this will work in combination with local fork tests. 
In the future, we might want to do similar fork testing like I did it for the `osx-plugin-template-hardhat`.
I guess we can adapt these functions once we decide to work on this.

Reply:
I think adding multiple approvers is a good improvement that should be added to the backlog for fork tests. I would avoid changing that inside this PR

Reply:
Agreed, can you add a task and link it here?